src/components/TextInput/TextInput.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../theme/fonts.scss';
+@import '../../theme/theme.scss';
+
+:root {
+  --tl-text-focused: var(--color-gs00);
+}
+
+.kai-text-input {
+  padding: 1rem;
+
+  &-label {
+    padding-bottom: 0.5rem;
+    display: block;
+    color: var(--color-epsilon);
+    font-size: 1.2rem;
+  }
+
+  &-input {
```

Comment:
Probably want a border around the input text when unfocused

Reply:
There's a border by default, but I guess your point here is that it doesn't quite match the spec?

Reply:
![image](https://user-images.githubusercontent.com/11202679/62195653-51e14d00-b331-11e9-8938-5457dc267aae.png)
Doesn't render a border on the textbox for me (Chrome on MacOS)

Reply:
Huh... I have been testing on Firefox 48 to match up with my phone. Anyway, I need to add the correct color (well, as close as I can get), so will do that 👍

Reply:
Awesome! I feel like with the heavy investments from Google, KaiOS may one day switch to a Chromium base rather than Firefox base. Tbh, I should actually be testing more in firefox than Chrome